### ISSUE TITLE
Create examples which shows how it works

### ISSUE THREAD

**Issue by supritashankar** [sabinaya/Hip-Game-Artificial-Intelligence#1]
Show examples how it works .. And how the input is given!

PS: Since exams are done :P

**Comment by sabinaya** [comment]
@supritashankar Yes. I will do that. I am going to code the game in python and convert into a web service. What say?

**Comment by supritashankar** [comment]
@sabinaya  Yes it sounds good. It will be much easier for you to write in Py.
